### DISCUSSION TITLE
I don't receive emails from WordPress<title>

### DISCUSSION THREAD

**Discussion by Awiem1234** [community/community#47744] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I'm receiving emails from everyone on my admin account except WordPress. Every order placed at my website was expected to be email at my admin email but no mail is received for any order. I'm unable to change email too as it ask for verification but I don't receive any verification email. What to do?

**Comment by queenofcorgis** [comment]
Hi @Awiem1234 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Wordpress. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by garousiamir** [comment]
If you are using woocommerce the admin email for getting woocommerce notifications can be managed through dashboard> woocommerce > options > email
if you are not using woocomerce and you want to change WordPress admin email without verification you can do that according to this tutorial:
https://cinchws.com/change-wordpress-admin-email-without-confirmation/
